File: client/iih_stub/iih.ci_stub/iih.ci.ord/cior/d/LastOfMpRstTp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.ord.cior.d
{
   public enum LastOfMpRstTp {
	/// <summary>
	/// 最后一顿
	/// </summary>
    [Description("最后一顿")]
    THELASTOFMP=1,
	/// <summary>
	/// 非最后一顿
	/// </summary>
    [Description("非最后一顿")]
    NOTTHELASTOFMP=2,
	/// <summary>
	/// 尚未计算
	/// </summary>
    [Description("尚未计算")]
    UNKNOWN=3,
   }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/cior/d/LastOfMpRstTp.java
package iih.ci.ord.cior.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface LastOfMpRstTp {

    @DmEnumDesc(name="最后一顿",description="最后一顿")
	public static final Integer THELASTOFMP=1; //最后一顿
    @DmEnumDesc(name="非最后一顿",description="非最后一顿")
	public static final Integer NOTTHELASTOFMP=2; //非最后一顿
    @DmEnumDesc(name="尚未计算",description="尚未计算")
	public static final Integer UNKNOWN=3; //尚未计算
}	
